Sources/Service/WalletAvailabilityService.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import Networking
+import UIKit
+
+
+public final class WalletAvailabilityService {
+    let defaults = UserDefaults.standard
```

Comment:
Not sure how much assigning this to a variable is giving us, might make it slightly harder to read

Reply:
That's fair.. I thought it was useful more so for the `WalletAvailabilityServiceTests`. But will remove for now & try to find a better alternative for the tests 😅